crates/test_util/build.rs

```diff
@@ -33,3 +38,73 @@ async fn download_open_jtalk_dict(out_dir: &Path) -> anyhow::Result<()> {
     dict_archive.unpack(out_dir)?;
     Ok(())
 }
+
+fn generate_snapshot_json(out_dir: &Path) -> anyhow::Result<()> {
```

Comment:
何か分からなくなりそうなのでちょっとドキュメント書くのお願いできると！
どう書くんでしたっけ･･･ `///`？

```suggestion
/// テストに用いるデータを作成する
fn generate_snapshot_json(out_dir: &Path) -> anyhow::Result<()> {
```